schema.json

```diff
@@ -0,0 +1,496 @@
+                                        "type": "object",
+                                        "required": [
+                                            "opts"
+                                        ],
+                                        "minProperties": 2,
+                                        "maxProperties": 2,
+                                        "propertyNames": {
+                                            "pattern": "^[a-zA-Z_]{1}[a-zA-Z0-9_]*$"
+                                        },
+                                        "properties": {
+                                            "opts": {
+                                                "type": "object",
+                                                "additionalProperties": false,
+                                                "properties": {
+                                                    "is_expand": {
+                                                        "type": "boolean"
+                                                    },
+                                                    "skip_if_empty": {
+                                                        "type": "boolean"
+                                                    },
+                                                    "title": {
+                                                        "type": "string"
+                                                    },
+                                                    "description": {
+                                                        "type": "string"
+                                                    },
+                                                    "summary": {
+                                                        "type": "string"
+                                                    },
+                                                    "category": {
+                                                        "type": "string"
+                                                    },
+                                                    "value_options": {
+                                                        "type": "array",
+                                                        "minProperties": 2,
+                                                        "items": {
+                                                            "type": "string"
+                                                        }
+                                                    },
+                                                    "is_required": {
+                                                        "type": "boolean"
+                                                    },
+                                                    "is_dont_change_value": {
+                                                        "type": "boolean"
+                                                    },
+                                                    "is_template": {
```

Comment:
if this is true then can we check if the value is a valid template?

Reply:
no, we can not set conditional checks